tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -47,7 +47,6 @@ Feature: Sharing files and folders with internal users
       | set-role    | expected-role | permissions-folder        | permissions-file |
       | Viewer      | Viewer        | read                      | read             |
       | Editor      | Editor        | read,change,create,delete | read,change      |
-      | Custom Role | Viewer        | read                      | read             |
```

Comment:
file and folder is tested, so if only the file share does not have any custom role, the test need to be splitted